Platform/CoffeelakeBoardPkg/Script/StitchIfwi.py

```diff
@@ -544,6 +544,10 @@ def main():
     ifwi_file_name = 'sbl_ifwi_%s.bin' % (args.platform)
     shutil.copy(generated_ifwi_file, ifwi_file_name)
 
+    generated_signed_sbl =  os.path.join(work_dir, 'Temp', 'SlimBootloader.bin')
+    sbl_file_name = 'SlimBootloader_%s.bin' % (args.platform)

```

Comment:
This patch would copy the signed SBL image to current folder. Late capsule image and other images might also be added by StitchIfwi.py. To avoid copy too many files to current folder, maybe we could copy the signed SBL image to other places. e.g. copy to Outputs\<platform> if the folder exists.
For the signed SBL image, maybe we could rename it sbl_<platform>_signed.bin?

Reply:
@gdong1 would you recommend to add new parameter for stitch outputs as "-outpath .."?

Reply:
Adding output path is also fine to me. then we could use current folder if  outpath is not provided.

Reply:
Yes added outpath option for user to specify.